shared/js/dataDriver.js

```diff
@@ -85,6 +85,18 @@ loop.DataDriver = function() {
     }
 
     /**
+      * Add a page to the room's table of contents.
+      *
+      * @param {String} url       The url of the page.
+      * @param {String} title     The title of the page.
+      * @param {Object} metadata  Additional metadata,
```

Comment:
Let me flip this question around: What's the purpose of having a single page object? Will there ever be any other keys at the top level? If not, it seems like a needless extra level of nesting.

Also presumably the url and title will be required but the metadata is always optional.